src/charm.py

```diff
@@ -38,7 +40,12 @@ def __init__(self, *args: typing.Any):
         self._database = DatabaseObserver(self, DATABASE_RELATION_NAME)
         self._matrix = MatrixObserver(self, MATRIX_RELATION_NAME)
         # 8090 is used for Synapse -> IRC Bridge communication
-        self.ingress = IngressPerAppRequirer(self, port=IRC_BIND_PORT, strip_prefix=True)
+        self.ingress = IngressPerAppRequirer(
+            self, port=IRC_BIND_PORT, strip_prefix=True, relation_name="ingress"
+        )
+        self.ingress_media = IngressPerAppRequirer(
```

Comment:
minor: should the relation name be a constant?

Reply:
Yes, I'll submit a refactor PR for that.